samples/DevTestLabs/SimpleUI/ClientApp/package-lock.json

```diff
@@ -0,0 +1,13687 @@
+{
+  "name": "simple_dtl_ui",
```

Comment:
Might consider removing this from the repo if it's a generated file

Reply:
Other Microsoft repos I've checked have included this file -- might make sense to ensure that users get the exact same set of dependencies when they try out the sample

Reply:
Sure, sounds good - if it's common to include it we should just leave it